test/unit/federation_server_test.js

```diff
@@ -227,5 +228,71 @@ FEDERATION_SERVER="https://api.stellar.org/federation"
           .then(() => tempServer.close());
       });
     });
+
+  });
+
+  describe('FederationServer times out when response lags and timeout set', function () {
+    let tempServer;
+    beforeEach(function() {
+      // Unable to create temp server in a browser
+      if (typeof window != 'undefined') {
+        return done();
+      }
```

Comment:
This has to be checked inside `it`s functions. `done` is not defined here. CI tests pass because we do not perform SauceLabs tests in PRs at it could leak our keys.

https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml
> Encrypted environment variables are not available to pull requests from forks due to the security risk of exposing such information to unknown code.

Reply:
ah, ok had to install firefox for this one